server/src/query/copy_move.ts

```diff
@@ -843,28 +844,12 @@ export async function checkIfContentContains({
   // Would it be better to do a single recursive query rather than recurse will individual queries
```

Comment:
`countAssigned` now does nothing. I assume that is intentional, as there is no other instance of `countAssigned` in the codebase. In that case, delete `countAssigned` from the arguments and docstring.